performance_manager/lib/postgres_utils.py

```diff
@@ -52,33 +93,6 @@ def postgres_create_modified_trigger(
         connection.execute(trigger_statement)
 
 
-def sqlite_create_modified_trigger(
-    table: sa.schema.Table,
-    connection: sa.engine.Connection,
-    **_: Any,
-) -> None:
-    """
-    sqlalchemy will listen for new table create events and when detected,
-    run this function to suplement table creation by creating update_on
-    triggers for tables containing an 'update_on' field
-
-    this function will only be used for sqlite db
-    """
-    update_column_name = "updated_on"
-    if update_column_name in list(table.columns.keys()):
-        trigger_name = f"update_{table}_modified".lower()
-        primary_key = str(table.primary_key.columns.items()[0][0])
-        trigger_statement = sa.DDL(
-            f"CREATE TRIGGER IF NOT EXISTS {trigger_name} "
-            f"AFTER UPDATE ON {table} FOR EACH ROW "
-            f"BEGIN "
-            f"  UPDATE {table} SET {update_column_name} = CURRENT_TIMESTAMP "
-            f"  WHERE {primary_key} = NEW.{primary_key};"
-            f"END; "
-        )
-        connection.execute(trigger_statement)
-
-
```

Comment:
got rid of all the sqlite stuff, don't think it's useful anymore and it's cluttering things up